app/src/main/java/org/simple/clinic/bp/assignbppassport/BpPassportSheet.kt

```diff
@@ -77,9 +83,27 @@ class BpPassportSheet : BottomSheetActivity(), BpPassportUiActions {
   }
 
   private val bpPassportIdentifier: Identifier by lazy {
-    intent.getParcelableExtra(KEY_BP_PASSPORT_NUMBER)!!
+    screenKey.identifier
+  }
+
+  override fun defaultModel() = BpPassportModel.create(bpPassportIdentifier)
+
+  override fun bindView(inflater: LayoutInflater, container: ViewGroup?): SheetBpPassportBinding {
+    return SheetBpPassportBinding.inflate(layoutInflater)
```

Comment:
We should use the version of the `inflate()` method that takes in the `container` parameter as well so that the layout inflater can set view attributes correctly.